lib/manageiq/appliance_console/database_admin.rb

```diff
@@ -213,6 +258,15 @@ def file_options
         end
       end
 
+      def api_version_menu_args
+        [
+          "OpenStack API Version",
+          API_VERSION_OPTIONS,
```

Comment:
Same thing here, I don't think this needs to be a constant.

Reply:
agreed.